meson.build

```diff
@@ -23,6 +23,9 @@ libavutil = dependency('libavutil', version: '>= 56.70.100')
 libswresample = dependency('libswresample', version: '>= 3.9.100')
 libswscale = dependency('libswscale', version: '>= 5.9.100')
 
+libplacebo = dependency('libplacebo', version: '>=6.292.0', fallback: ['libplacebo', 'libplacebo'],
```

Comment:
I still don't understand, why `fallback: ['libplacebo', 'libplacebo']` twice?

Reply:
https://mesonbuild.com/Subprojects.html#toggling-between-system-libraries-and-embedded-sources

Subprojects can't be used as dependencies, it has to take the libplacebo variable from libplacebo's meson.build as the dependency as far as I understand.

Reply:
Thanks, I see now. `['subproject_name', 'variable_name']`, great syntax. Not that they don't have dicts...